adminSiteClient/ImagesIndexPage.tsx

```diff
@@ -59,11 +85,89 @@ function AltTextEditor({
     )
 }
 
+function UserSelect({
+    usersMap,
+    initialValue = "",
+    onUserSelect,
+}: {
+    usersMap: UserMap
+    initialValue?: string
+    onUserSelect: (user: DbPlainUser) => void
+}) {
+    const [isSetting, setIsSetting] = useState(false)
+
+    const { admin } = useContext(AdminAppContext)
+    const [value, setValue] = useState(initialValue)
+    const [filteredOptions, setFilteredOptions] = useState(() =>
+        mapToArray(usersMap).map((user) => ({
+            value: user.fullName,
+            label: user.fullName,
+        }))
+    )
+
+    const handleChange = (value: string) => {
+        setValue(value)
+        const lowercaseValue = value.toLowerCase()
+        setFilteredOptions(
+            mapToArray(usersMap)
+                .filter((user) =>
+                    user.fullName.toLowerCase().includes(lowercaseValue)
+                )
+                .map((user) => ({
+                    value: user.fullName,
```

Comment:
I don't think we really need the map instead of an array, but since we already have it, should the value be the ID so we can then look the user up by it in `handleSelect`?